src/com/sony/simdetect/BootCompletedReceiver.kt

```diff
@@ -14,19 +14,24 @@
  * limitations under the License.
  */
 
-package com.sony.simdetect;
+/*
+ * Rewritten in Kotlin by Pavel Dubrova <pashadubrova@gmail.com>
+ */
 
-import android.content.BroadcastReceiver;
-import android.content.Context;
-import android.content.Intent;
-import android.util.Log;
+package com.sony.simdetect
 
-public class BootCompletedReceiver extends BroadcastReceiver {
-    private static final String TAG = "SimDetect";
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+
+class BootCompletedReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent?) {
+        Log.d(TAG, "Starting")
+        context.startService(Intent(context, SimDetectService::class.java))
+    }
 
-    @Override
-    public void onReceive(final Context context, Intent intent) {
-        Log.d(TAG, "Starting");
-        context.startService(new Intent(context, SimDetectService.class));
+    companion object {
```

Comment:
is `companion object` supposed to be on the bottom or top? It'd be cool if you could unify both *.kt files.
Also you can update copyrights to 2019-2020 and I wouldn't mind if you put same copyright (as in LineageOS) in Android.bp but you don't have to if you're against that idea.

Reply:
done